actionlist/app/components/action_item.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableHighlight,
+} from 'react-native';
+import ActionItemBody from './action_item_body';
+import { toggleItem } from '../actions/action_items';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#e9e9e9',
+    borderBottomWidth: 0.5,
+    paddingRight: 10,
+    overflow: 'hidden',
+  },
+  titleContainer: {
+    flexDirection: 'row',
+  },
+  title: {
+    flex: 1,
+    padding: 10,
+    color: '#7B1500',
+    fontWeight: 'bold',
+  },
+  buttonImage: {
+    marginTop: 10,
+    width: 20,
+    height: 20,
+  },
+  body: {
+    padding: 10,
+    paddingTop: 0,
+  },
+});
+
+const icons = {
+  /* eslint-disable global-require */
```

Comment:
Please change these to single line `disables` (or re-enable them after these two `require`s).